### DISCUSSION TITLE
Cannot load oh-my-posh in clink due to spaces in oh-my-posh.exe path

### DISCUSSION THREAD

**Discussion by wallali** [JanDeDobbeleer/oh-my-posh#1538] [Troubleshoot]
```
--oh-my-posh.lua
load(io.popen('oh-my-posh --config="C:/Users/jan/jandedobbeleer.omp.json" --init --shell cmd'):read("*a"))()
```

This suggested way to enable oh-my-posh does not work for me because `oh-my-posh.exe` is located at a path with spaces, eg.

`C:\Users\jan\OneDrive\OneDrive - Jan\Documents\WindowsPowerShell\Modules\oh-my-posh\oh-my-posh.exe`

with error

`'C:/Users/jan/OneDrive/OneDrive' is not recognized as an internal or external command, operable program or batch file.`

**Comment by JanDeDobbeleer** [comment]
using a properly escaped path should work without issues, I would expect that Lua would be able to handle that.

**Reply by wallali** [reply]
hmm, not quite sure how to escape it, I have tried double quotes `"`, square brackets `[[]]` etc. It's not as straightforward because the quoted string then leads to other errors (eg `cmd` unable to resolve `"oh-my-posh"` as an executable for eg). Would appreciate some help.

`'oh-my-posh" --config="C:' is not recognized as an internal or external command, operable program or batch file.`

ref: https://stackoverflow.com/questions/53452818/lua-io-popen-run-program-with-space-in-path
